github/Repository.py

```diff
@@ -1448,6 +1487,162 @@ def __create_pull(self, **kwds):
             self._requester, headers, data, completed=True
         )
 
+    def create_repository_advisory(
+        self,
+        summary: str,
+        description: str,
+        severity_or_cvss_vector_string: str,
+        cve_id: typing.Optional[str] = None,
+        vulnerabilities: typing.Optional[
+            typing.Iterable[
+                github.RepositoryAdvisoryVulnerability.AdvisoryVulnerability
+            ]
+        ] = None,
+        cwe_ids: typing.Optional[typing.Iterable[str]] = None,
+        credits: typing.Optional[
+            typing.Iterable[github.RepositoryAdvisoryCredit.RepositoryAdvisoryCredit]
+        ] = None,
+    ) -> github.RepositoryAdvisory.RepositoryAdvisory:
+        """
+        :calls: `POST /repos/{owner}/{repo}/security-advisories <https://docs.github.com/en/rest/security-advisories/repository-advisories>`_
+        :param summary: string
+        :param description: string
+        :param severity_or_cvss_vector_string: string
+        :param cve_id: string
+        :param vulnerabilities: iterable of :class:`github.RepositoryAdvisoryVulnerability.AdvisoryVulnerability`
+        :param cwe_ids: iterable of string
+        :param credits: iterable of :class:`github.RepositoryAdvisoryCredit.RepositoryAdvisoryCredit`
+        :rtype: :class:`github.RepositoryAdvisory.RepositoryAdvisory`
+        """
+        return self.__create_repository_advisory(
+            summary=summary,
+            description=description,
+            severity_or_cvss_vector_string=severity_or_cvss_vector_string,
+            cve_id=cve_id,
+            vulnerabilities=vulnerabilities,
+            cwe_ids=cwe_ids,
+            credits=credits,
+            private_vulnerability_reporting=False,
+        )
+
+    def report_security_vulnerability(
```

Comment:
I think this should read:
```suggestion
    def report_security_advisory(
```
or
```suggestion
    def create_security_advisory(
```

Reply:
This API is used by reporters, not by maintainers, to report vulnerabilities.

I was trying to figure out a way to differentiate the `create` which is something only a repository admin can take, from the `report` which is an action any GitHub user can take.

Reply:
right, the link to the docs confused me, I think it is linking to creating an advisory, where it should link to creating a vulnerability